portfolio/src/main/java/com/google/sps/servlets/DeleteContactServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/delete-contact")
```

Comment:
Please add a comment outlining the purpose of this class.

Reply:
okay

Reply:
*This
Also, Same comment as above w.r.t. line length.

Reply:
Was this updated?

Reply:
Please address this.